### DISCUSSION TITLE
WooCommerce "Main" Template File.

### DISCUSSION THREAD

**Discussion by DLX23** [holger1411/understrap#1567] [Q&A]
Hi Guys,

where i can find the main woocommerce template fail that is used by /shop ?

**Comment by bacoords** [comment]
I recommend checking the Woocommerce documentation for more complete info on their templates: https://woocommerce.com/document/template-structure/

I also recommend running the Query Monitor plugin on your dev environment: http://wordpress.org/plugins/query-monitor/ This plugin can typically tell you every php template file loaded on whatever screen you're looking it. It's VERY helpful to see where the php is coming from.
